database/snowflake/README.md

```diff
@@ -0,0 +1,9 @@
+# Snowflake
+
+`snowflake://user:password@accountname/schema/dbname?query`
+
+| URL Query  | WithInstance Config | Description |
+|------------|---------------------|-------------|
+| `x-migrations-table` | `MigrationsTable` | Name of the migrations table |
+
+Snowflake is PostgreSQL compatible but has some specific features (or lack thereof) that require slightly different behavior.
```

Comment:
Add a note about Snowflake not being officially supported since there aren't any tests.

If you'd like, you can also link to this driver in top-level README, but you don't have to.

Reply:
Added a line saying that Snowflake cant be made to run locally.